layouts/shortcodes/pages/compatibility-products.html

```diff
@@ -8,47 +8,120 @@
+          <span class="icon-bar"></span>
+        </div>
+      </button>
+    </div>
   </div>
-  <div class="col-md-8" style="text-align: center;"><a class="btn btn-lg btn-secondary" href="https://github.com/eclipse-ee4j/jakartaee-platform/issues?q=is%3Aissue+is%3Aclosed+label%3Acertification">Compatibility Requests</a></div>
+</div>
+ 
+<div class="row">
+ <div class="navbar-collapse collapse" id="navbar-member-levels">
+   <ul class="nav nav-tabs solstice-tabs content-nav-tab" role="tablist">
+     <li role="presentation" class="active">
+       <a class="background-grey" href="#tab-one" data-toggle="tab" aria-controls="tab-one" role="tab">
+        Jakarta EE Compatible Products
+       </a>
+     </li>
+     {{ range $.Site.Data.compatible_products.sets }}
+     <li role="presentation" class="">
+       <a class="background-grey" href="#tab-{{ .jakartaee_version }}" data-toggle="tab" aria-controls="tab-{{ .jakartaee_version }}" role="tab">
+        Jakarta EE {{ .jakartaee_version }}
+       </a>
+     </li>
+     {{ end }}
+   </ul>
+ </div>
 </div>
 
-{{ range $.Site.Data.compatible_products.sets }}
-  {{ $title:=.title }}
-  <div class="product-list">
-    <h2 class="text-center heading-line" id="{{ .title | urlize }}"><span style="background-color:#fff;">{{ .title }}</span></h2>
+<div class="content-nav-tab-body tab-content margin-bottom-20" id="tabs-content">
+
+  <div role="tabpanel" class="tab-pane tab-pane active" id="tab-one">
+    {{/*  <h2 class="h3 margin-top-0">Jakarta EE Compatible Products</h2>  */}}
+    {{ $new_product_list := slice }}
+    {{ range $elem_index, $elem_val := $.Site.Data.compatible_products.sets }}
+      {{ range $elem_val.items }}
+        {{ range .items }}  <!-- Loop all items and add name, image, image_width properties to form a new object and put in the new_product_list array -->
+          {{ $new_element := (dict "name" .name "image" .image "image_width" .image_width) }}
+          {{ $new_product_list = $new_product_list | append $new_element }}
+        {{ end }}
+      {{ end }}
+    {{ end }}
+
     <div class="row">
-      {{ range sort .items "name" "asc"}}
-        <div class="col-xs-24 col-sm-12 col-lg-6 padding-bottom-20">
-          <div class="match-height-item-by-row padding-20 flex-column">
-            <div class="center-block padding-left-30 padding-right-30 padding-bottom-10 padding-top-10 grid-center" data-mh="{{ $title | urlize }}">
-              {{ if .image }}
-              <a href="{{ .download }}"><img width="{{ .image_width }}" class="img img-responsive" src="{{ .image }}" alt="{{ .vendor }}'s {{ .name }} logo"></a>
-              {{ else }}
-	            <div class="icon-backdrop">
-	              <div class="ratio"></div>
-	              <div class="icon-container"><i class="fa fa-download"></i></div>
-	            </div>
-              {{ end }}
+      {{ range sort ($new_product_list | uniq) "name" "asc" }}  <!-- Remove duplicates and re-order by name -->
```

Comment:
This could delete "duplicates" that are just versions across different versions of Jakarta EE. Have you verified this case?

Reply:
That's my intent, since on the first tab, we just display all logos, right? we cannot put two same logos 🤔

Reply:
I haven't really looked into the design for this yet. I'm just making sure that this is all intentional :)